### DISCUSSION TITLE
nvim-lightbulb in sign column doesn't show up

### DISCUSSION THREAD

**Discussion by kapral18** [LazyVim/LazyVim#1562] [Q&A] [answered]
Could anyone share a working config of lightbulb integration working with default sign column. It works for virtual_text setup but doesn't work for default sign column setup. 

my config https://github.com/kapral18/config.nvim/blob/ffdb572892a01f74b7328cc8279008136258baf0/lua/plugins/improve-code-actions.lua

**Comment by dpetka2001** [comment]
I tried your configuration with a fresh Lazyvim starter template with default configuration and only adding the snippet of code for `nvim-lightbulb` and it worked for me.

**Reply by kapral18** [reply]
weird because default sign column doesn't show for me, only virtual_text to the right of text shows. But thanks for checking

**Reply by dpetka2001** [reply]
You might want to check if it works with a fresh Lazyvim starter template. If it does, then it must be something in your current configuration.

**Reply by dpetka2001** [reply]
Other thing I could think of is maybe it has to do with your Neovim version. I use nightly and don't observe this issue.

**Reply by jyuan0** [reply]
@kapral18 it might have to do with the recent LazyVim `statuscolumn` changes. You could try setting `statuscolumn` back to the default (`set statuscolumn=`) to see if it shows up, but you'll lose the recent `statuscolumn` improvements.

**Reply by dpetka2001** [reply]
@jyuan0 Good catch. I can confirm that with the latest changes I also don't get the lightbulb, whereas I used to get it before(I must have tried it with a Lazyvim version before the `statuscolumn` changes because I can see that `statuscolumn` changes were 2 days ago but the issue was opened 3 days ago). Setting `statuscolumn` to default made the icon appear. @folke I'm sorry for pinging you, but would you mind taking a look at this if it's something related to Lazyvim or is it related to the plugin itself?

**Reply by folke** [reply]
Looking into it. I opened an issue https://github.com/LazyVim/LazyVim/issues/1596

**Reply by kapral18** [reply]
@jyuan0 @dpetka2001 I really appreciate your input guys, thank you!

**Comment by folke** [comment, marked as answer]
Should be fixed now. Check on LazyVim dev or wait for the new release (tomorrow).

Be aware that only the highest priority sign will be shown in the left column.
The default sign priority for lighbulb is 10, while warnings (12) and errors are 13.

So on lines with diagnostics, the lightbulb will not be shown in favor of the diagnistics.

lightbulb.nvim has a priority setting that you can change to always show the lighbulb if available. 

See also https://github.com/LazyVim/LazyVim/issues/1596

**Reply by kapral18** [reply]
thanks so much for your work
